### PR TITLE
Update docs for latest gtm v3 tag and variable

### PR DESCRIPTION
**Pull request by adatzer**
Updates after releasing [Snowplow v3](https://github.com/snowplow/snowplow-gtm-tag-template-v3/releases/tag/1.1.0) tag and [Snowplow v3 Settings](https://github.com/snowplow/snowplow-gtm-variable-template-v3/releases/tag/1.2.0) variable.

### CODE REVIEW

**Review by AlexBenny — APPROVED** [review]
👍  LGTM!! 
Feel free to ask anyone with more GTM experience btw if you need an expert eye.

File: docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/google-tag-manager-custom-template/template-for-javascript-tracker-v3/tag-template-guide/index.md [hunk 1]
```diff
@@ -4,27 +4,27 @@ date: "2021-11-18"
...
 
 ## Caveats
 
 To begin with, some of the caveats of using the Custom Template.
 
 - Any methods that require the parsing of HTML elements (e.g. link tracking filter functions, cross domain linking) will not work and are thus disabled.
 - Automatic error tracking does not work due to lack of support for the `ErrorEvent` API.
-- There is no implementation for the [standard ecommerce](https://github.com/snowplow/snowplow/wiki/2-Specific-event-tracking-with-the-Javascript-tracker#ecommerce) events. Users are encouraged to implement the [enhanced ecommerce](https://github.com/snowplow/snowplow/wiki/2-Specific-event-tracking-with-the-Javascript-tracker#enhanced-ecommerce) setup instead.
+- There is no implementation for the [standard ecommerce](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracking-events/#ecommerce-tracking) events. Users are encouraged to implement the [enhanced ecommerce](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracking-events/#enhanced-ecommerce-tracking) setup instead.
```
Comment (stanch) [inline docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/google-tag-manager-custom-template/template-for-javascript-tracker-v3/tag-template-guide/index.md:27]: Use a markdown link please (see https://github.com/snowplow/snowplow.github.io#links)

Comment (stanch) [reply docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/google-tag-manager-custom-template/template-for-javascript-tracker-v3/tag-template-guide/index.md:27]: (here and below)

Comment (stanch) [reply docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/google-tag-manager-custom-template/template-for-javascript-tracker-v3/tag-template-guide/index.md:27]: You can try running `./make-links-validatable.sh` to do that automatically

**Review by stanch — CHANGES_REQUESTED** [review]

**Review by igneel64 — APPROVED** [review]
LGTM

File: docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/google-tag-manager-custom-template/index.md [hunk 2]
```diff
@@ -4,13 +4,4 @@ date: "2020-08-10"
...
-
-There are two templates that you will need to use when setting up the Snowplow Custom Template.
-
-1. Snowplow Analytics Settings
-2. Snowplow Analytics
-
-Snowplow Analytics Settings Template: A variable template which can be used to easily apply a set of tracker configuration parameters to tags created with the Snowplow Analytics tag template.
-
-Snowplow Analytics Template: Load, configure, and deploy the Snowplow JavaScript tracker library. It supports the full functionality of the JavaScript SDK.
+The Snowplow JavaScript Tracker can now be deployed directly to Google Tag Manager via a Custom Template.
```
[the remark this answers is not in the document]
Comment (adatzer) [reply docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/google-tag-manager-custom-template/index.md]: This is indeed a bit short but i am not sure what else to say at this point without repeating things of the inner pages. I tried to rephrase the sentence a bit and added the cards, hoping it makes it at least look nicer. Do you think this is ok?

**Review by stanch — APPROVED** [review]

**Pull request merged**